fhir-term-graph/src/main/java/com/ibm/fhir/term/graph/factory/FHIRTermGraphFactory.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * (C) Copyright IBM Corp. 2020
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.ibm.fhir.term.graph.factory;
+
+import org.apache.commons.configuration.Configuration;
+import org.apache.commons.configuration.PropertiesConfiguration;
+
+import com.ibm.fhir.term.graph.FHIRTermGraph;
+import com.ibm.fhir.term.graph.impl.FHIRTermGraphImpl;
+
+public final class FHIRTermGraphFactory {
+    private FHIRTermGraphFactory() { }
+
+    public static FHIRTermGraph open(String propFileName) {
+        try {
+            return open(new PropertiesConfiguration(propFileName));
```

Comment:
I might of missed where this configuration needs to be configured
Maybe a description in the class or the corresponding guide?

Reply:
Adding Javadoc